pkg/eks/api.go

```diff
@@ -34,10 +22,25 @@ import (
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
 	awseks "github.com/aws/aws-sdk-go/service/eks"
 	"github.com/aws/aws-sdk-go/service/eks/eksiface"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/elb/elbiface"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/aws/aws-sdk-go/service/elbv2/elbv2iface"
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/aws/aws-sdk-go/service/iam/iamiface"
 	"github.com/aws/aws-sdk-go/service/sts"
 	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+	"github.com/kris-nova/logger"
```

Comment:
We tend to have the following kind grouping:

- standard library
- anything from github (includes: logger, errors)
- AWS SDK
- Kubernetes
- Weaveworks

Reply:
The problem with that is that:
1. It's not really respected in general (I've run into a few examples in which the `eksctl` dependencies were not at the end)
2. You cannot automate it with an IDE (at least not easily). At least GoLand only lets you organize by:
```
Stdlib

external dependencies

local dependencies
```

I am pretty sure (1) is most probably due to (2)

Also (and this is just a personal opinion) I don't see the use of it. How is this helpful to you?

Reply:
Also remember about our external contributors. I think whichever we decide it should be as standard as possible.